maui-toolkit/Bottom-Sheet/BottomSheet-Customization.md

```diff
@@ -17,20 +17,101 @@ The [IsModal](https://help.syncfusion.com/cr/maui-toolkit/Syncfusion.Maui.Toolki
 {% tabs %}
 {% highlight xaml %}
 
-<bottomSheet:SfBottomSheet x:Name="bottomSheet" IsModal="True">
+<bottomSheet:SfBottomSheet x:Name="bottomSheet" IsModal="True" HalfExpandedRatio="0.35" ContentPadding="10">
     <bottomSheet:SfBottomSheet.BottomSheetContent>
-        <Grid/>
+        <VerticalStackLayout Spacing="5" x:Name="bottomSheetContent">
```

Comment:
No need to add content here

Reply:
@naveenkumar-sanjeevirayan,

Will update as discussed.